src/app/app.module.ts

```diff
@@ -1,11 +1,13 @@
 import { Module } from '@nestjs/common';
-import { GraphQLModule } from '@nestjs/graphql';
-import { ConfigModule } from 'protocol-common/config.module';
-import { AppService } from './app.service';
-import { AppController } from './app.controller';
-import data from '../config/env.json';
-import { OrmConfig } from '../ormconfig';
-import { ReportModule } from '../report/report.module';
+import pkg from '@nestjs/graphql';
+const { GraphQLModule } = pkg;
```

Comment:
Similar comment to the `ObjectType` comment on `report.dto.ts`

Reply:
result same as above, unfortunately

Reply:
Yeah, unfortunately that doesn't work. But I do think `pkg` should be renamed to `nestJsGraphql` or something similar.